### PR TITLE
API-9878: License/organization lookup in API v3

### PR DESCRIPTION
**Pull request by grz-gajda**

### CODE REVIEW

File: configuration/api_test.go [hunk 1]
```diff
@@ -271,12 +274,22 @@ func createMockedResponder(t *testing.T, method string) roundTripFunc {
 			}
 		}
 
-		if req.URL.String() != "https://api.livechatinc.com/v3.4/configuration/action/"+method {
+		expectedMethod := http.MethodPost
```
Comment (gluszczyk) [inline configuration/api_test.go:277]: `expectedHttp` por favor, let's not use the same name (method) for two different things

File: configuration/api_test.go [hunk 2]
```diff
@@ -271,12 +274,22 @@ func createMockedResponder(t *testing.T, method string) roundTripFunc {
...
+		if method == "get_organization_id" || method == "get_license_id" {
+			expectedMethod = http.MethodGet
+		}
+
+		url := "https://api.livechatinc.com/v3.4/configuration/action/" + method
+		if expectedMethod == http.MethodGet && !strings.Contains(req.URL.String(), url) {
```
Comment (gluszczyk) [inline configuration/api_test.go:283]: maybe `HasPrefix` is better suited here, as we need req URL to start with expected URL?

File: configuration/api.go [hunk 3]
```diff
@@ -466,3 +466,17 @@ func (a *API) ListAutoAccesses() ([]*AutoAccess, error) {
 	err := a.Call("list_auto_accesses", &listAutoAccessesRequest{}, &resp)
 	return resp, err
 }
+
+// GetOrganizationID returns organization ID by given license ID.
```
[the remark this answers is not in the document]
Comment (grz-gajda) [reply configuration/api.go]: Fixed.

File: configuration/api_test.go (hunk 1 above)
Comment (grz-gajda) [reply configuration/api_test.go]: Renamed to `expectedHTTPMethod`.

File: configuration/api_test.go (hunk 2 above)
Comment (grz-gajda) [reply configuration/api_test.go]: Right, `HasPrefix` maybe the better choice.

**Review by gluszczyk — APPROVED** [review]

**Pull request merged**